packages/bpk-component-journey-arrow/src/BpkJourneyArrow.module.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use '../../../packages/unstable__bpk-mixins/tokens';
+@use '../../../packages/unstable__bpk-mixins/icons';
+
+.bpk-journey-arrow {
+  display: flex;
+  width: 100%;
+  min-width: tokens.bpk-spacing-xxl();
+  height: tokens.bpk-spacing-md();
+  margin: 0 tokens.bpk-spacing-md();
+  justify-content: center;
+  border-width: 0 5 * tokens.$bpk-one-pixel-rem 0 2 * tokens.$bpk-one-pixel-rem;
+  border-style: solid;
+  border-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="8" fill="none" viewBox="0 0 16 8"><path fill="%23C1C7CF" d="M15.69 4.35a.5.5 0 0 0 0-.7L12.51.46a.5.5 0 1 0-.7.71L14.62 4 11.8 6.83a.5.5 0 0 0 .7.7l3.2-3.17m-.7.14v-1H.5a.5.5 0 0 0 0 1Z"/></svg>')
```

Comment:
Will double check how we do this in other parts of the codebase to see if this is the right way or if there is a better way :)

Reply:
Yeah this is the spicy bit 😄 I see other URL SVG references in the code base, using the base64 encoding instead. But no other uses of border-image. As per https://caniuse.com/border-image this is widely supported though.